community/community-leaders.md

```diff
@@ -33,6 +33,8 @@ The technical steering committee (TSC) is accountable to the Project Governance
 
 On October 27, 2021, the following people were voted in as the TSC through October 2022 and the date of the next election's results.
 
+During Summer 2021, former Co-Chair of the Baseline TSC John Wolpert (Consensys Mesh) stepped down for personal reasons. Sonal Patel was elected to serve the remainder of the term in his place.
```

Comment:
not sure if we should state this, and also why do we have to state personal reasons?

Reply:
Noted. I agree most clear option may be to remove this wording altogether.